doc/classes/DirAccess.xml

```diff
@@ -253,7 +253,46 @@
 			<param index="0" name="path_a" type="String" />
 			<param index="1" name="path_b" type="String" />
 			<description>
-				Returns [code]true[/code] if paths [param path_a] and [param path_b] resolve to the same file system object. Returns [code]false[/code] otherwise, even if the files are bit-for-bit identical (e.g., identical copies of the file that are not symbolic links).
+				Returns [code]true[/code] if paths [param path_a] and [param path_b] resolve to the same file system object (taking into account case sensitivity, relative paths, symbolic links, and hard links). Returns [code]false[/code] otherwise, even if the files are exactly identical (copies of a file that are not symbolic links).
+				[codeblocks]
+				[gdscript]
+				var dir = DirAccess.open("/")
+				# Prints "true" on a case-insensitive filesystem, "false" otherwise.
+				print(dir.is_equivalent("/sbin/md5sum", "/sBin/MD5Sum"))
+
+				var icon_1_path = ProjectSettings.globalize_path("res://icon.svg")
+				var icon_2_path = ProjectSettings.globalize_path("res://icon2.svg")
+
+				dir.create_link(icon_1_path, icon_2_path)
+				# Prints `true`, icon_2_path points to a symbolic link.
```

Comment:
Not going to highlight every occurrence of this but they should be surrounded by quotation marks.
```suggestion
				# Prints "true", icon_2_path points to a symbolic link.
```